src/language/utils.ts

```diff
@@ -77,7 +77,7 @@ export function scanAllTokens(scanner: ts.Scanner, callback: (s: ts.Scanner) =>
 /**
  * @returns true if any modifier kinds passed along exist in the given modifiers array
  */
-export function hasModifier(modifiers: ts.ModifiersArray, ...modifierKinds: ts.SyntaxKind[]) {
+export function hasModifier(modifiers: ts.ModifiersArray|null|undefined, ...modifierKinds: ts.SyntaxKind[]) {
```

Comment:
you're right, fixed
I also noticed, that modifierKinds can never be null, so I changed the condition to check for `.length === 0`